src/devices/wholeBodyDynamics/WholeBodyDynamicsDevice.cpp

```diff
@@ -638,6 +680,14 @@ bool WholeBodyDynamicsDevice::loadSettingsFromConfig(os::Searchable& config)
         portPrefix = prop.find("portPrefix").asString();
     }
 
+    // Enable or disable ft streaming. The default value is false;
+    // is set in the device constructor
+    if( prop.check("streamFilteredFT") &&
+        prop.find("streamFilteredFT").isBool())
```

Comment:
Please document this new parameter in https://github.com/robotology/codyco-modules/blob/be0ebc82fe6f6c2c087263b2a001547cdec25eb1/src/devices/wholeBodyDynamics/WholeBodyDynamicsDevice.h#L118 .

Reply:
Done in a83c188a98e2bd4f75f58a484361ed2a88fffea7